From 5c71a4c98ce8afc07acadab188ad4ff51dc9e1d6 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Thu, 14 Apr 2011 13:24:02 +0800
Subject: ENGR00142005-1 ARM: imx: srtc: the end address minus 1

The end address of SRTC device in i.MX platform should be minus
1. Otherwise, it conflicts with the address of LDB device

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/plat-mxc/devices/platform-imx_srtc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/devices/platform-imx_srtc.c b/arch/arm/plat-mxc/devices/platform-imx_srtc.c
index 7dd2305..1f27f48 100755
--- a/arch/arm/plat-mxc/devices/platform-imx_srtc.c
+++ b/arch/arm/plat-mxc/devices/platform-imx_srtc.c
@@ -39,7 +39,7 @@ struct platform_device *__init imx_add_srtc(
 	struct resource res[] = {
 		{
 			.start = data->iobase,
-			.end = data->iobase + SZ_16K,
+			.end = data->iobase + SZ_16K - 1,
 			.flags = IORESOURCE_MEM,
 		}, {
 			.start = data->irq,
-- 
1.7.9.5

